src/components/Details.svelte

```diff
@@ -74,13 +75,16 @@
             </NotEmpty>
         </div>
 
+        <MarkdownField title={$_('details.notes')} content={$details.notes}/>
 
-        <NotEmpty value={$details.notes}>
-            <p><strong>{$_('details.notes')}: </strong>{@html marked($details.notes)}</p>
-        </NotEmpty>
-
-        {#if $details.buyagiftcardtosupportyourfavoriteevmerchantorganization || $details.supportyourfavoriteevmerchantorganizationsfundraiser}
+        {#if $details.buyagiftcardtosupportyourfavoriteevmerchantorganization ||
+            $details.supportyourfavoriteevmerchantorganizationsfundraiser ||
+            $details.giftcardfundraiser ||
+            $details.specialoffers
+        }
             <hr>
+            <MarkdownField title={$_('details.giftcardfundraiser')} content={$details.giftcardfundraiser}/>
```

Comment:
If `$details.giftcardfundraiser` is empty, do we still want to render this component instance? Similar question for the next line.

Reply:
Nope. I will add something to component to check for empty field.

Reply:
If we don't want to render anything, then you could add a condition around the component instance:

```
{#if $details.specialoffers}
  <MarkdownField title={$_('details.specialoffers')} content={$details.specialoffers}/>
{/if}
```

This pattern will result in more code since there will be an if block wherever you use the component instance, compared to a single if / return null in the component. However, I think it's preferable because it shows you that sometimes nothing is rendered for this component without having to look at the underlying component code.